src/shared/components/layout/Header/Header.module.scss

```
@@ -18,15 +18,14 @@
 .header {
   z-index: 100;
   position: fixed;
-  top: 0;
-  left: 0;
   width: 100%;
   @include m.flex-box(space-between, center, row);
   padding: 6px 8px;
   background-color: f.color(white);
   border-bottom: 1px solid #e5e7eb;
   box-shadow: 0 1px 3px rgba(0, 0, 0, 0.06);
```

Stylelint (17.2.0)

[error] Expected "0.06" to be "6%" (alpha-value-notation)

(alpha-value-notation)